lms/static/lms/js/require-config.js

```diff
@@ -100,6 +100,7 @@
             'string_utils': 'js/src/string_utils',
             'utility': 'js/src/utility',
             'draggabilly': 'js/vendor/draggabilly',
+            'which-country': 'common/js/vendor/which-country',
```

Comment:
Were you not able to use npm to install it? Not a big deal, but it would be great to pull it in that way instead of copying it to ```vendor```

Reply:
removed